elasticsearch/run.sh

```diff
@@ -85,7 +85,7 @@ wait_for_port_open() {
 do
     echo -n "."
     sleep $RETRY_INTERVAL
-    (( retry -= 1 ))
+    (( retry -= 1 )) || :
```

Comment:
The `expr` command, and numeric evaluations done inside `(( .... ))`, if they return a value of `0`, will be interpreted by the shell as an error return.  Since we run with `set -e`, this will cause the script to exit with an error.  Since the only way these numeric operations can fail in these cases is to return `0` (as opposed to say a divide by 0), the fix is to just ignore any errors returned from these by using `|| :` which means `if the previous command returned an error, do nothing`.